app/components/Map.js

```diff
@@ -13,7 +13,7 @@ export default class Map extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      position: null,
+      // position: null,
```

Comment:
I noticed this state.position does not do anything, so I was testing it out. I want to take it out in a new PR that deals with that proptype bug